### PR TITLE
[shipui-1926] Changes: Consolidated shippingLabel orderItem and packageRequest with Package PackageRequest

### PR DESCRIPTION
**Pull request by DimitarChobanovSellerCloud**

### CODE REVIEW

File: src/SellerCloud.Rest/ExactShip/Requests/Packages/OrderItemRequest.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
-﻿namespace SellerCloud.Rest.ExactShip.Requests.ShippingLabel
+﻿namespace SellerCloud.Rest.ExactShip.Requests.Packages
 {
-    public class ShippingLabelOrderItemRequest
+    public class OrderItemRequest
```
Comment (stanimirsellercloud) [inline src/SellerCloud.Rest/ExactShip/Requests/Packages/OrderItemRequest.cs:3]: Thanks for consolidating these models.

Can you please confirm if we use `OrderItemRequest` and `PackageRequest` for the **Rates** feature-related request? I believe this is where the task originated from.

Comment (DimitarChobanovSellerCloud) [reply src/SellerCloud.Rest/ExactShip/Requests/Packages/OrderItemRequest.cs:3]: Yes they are used in all Rates requests and that is where the task originated from. I took in account that and the changes to these models should be what is required for rates and shipping end-points.

**Review by stanimirsellercloud — APPROVED** [review]
Thanks @DimitarChobanovSellerCloud

File: src/SellerCloud.Rest/ExactShip/Requests/Packages/OrderItemRequest.cs (hunk 1 above)
Comment (stanimirsellercloud) [reply src/SellerCloud.Rest/ExactShip/Requests/Packages/OrderItemRequest.cs:3]: Thanks for the details.

**Pull request merged**